src/components/content/content.ts

```diff
@@ -782,6 +785,11 @@ export class Content extends Ion implements OnDestroy, AfterViewInit, IContent {
       this._cBottom += this._tabbarHeight;
     }
 
+    // Refresher uses a border which should be hidden unless pulled
+    if (this._hasRefresher === true) {
```

Comment:
@manucorporat Haha, I like the explicitness and it could be a tiny bit faster but sure ;)